session/container.go

```diff
@@ -376,3 +376,17 @@ func (x Container) AssertAuthKey(key neofscrypto.PublicKey) bool {
 
 	return bytes.Equal(bKey, x.body.GetSessionKey())
 }
+
+// IsOwnedBy returns true if session token is signed
+// and, therefore, owned by specified user.
+//
+// See also Sign.
+func (x Container) IsOwnedBy(id user.ID) bool {
```

Comment:
Correct, but what do u think about at the name `IssuedBy`? There are some similar words in `SetIat` docs.

A simple unit test would be also appreciated.

Reply:
Renamed, unit test had been added too.